src/components/__tests__/SignupAlert.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import Alert from '../SignupAlert';
+
+describe('<Alert/>', () => {
+  describe('render', () => {
+    const signupdata = {
+      user: {
+        email: 'dummy@gmail.com',
+        username: 'dummy'
+      }
+    };
+
+    const erroredMail = {
+      user: {},
+      errors: {
+        data: {
+          errors: {
+            email: ['The email has been used']
+          }
+        }
+      }
+    };
+
+    test('renders alert component on success', () => {
+      const wrapper = shallow(
+        <Alert
+          signupdata={signupdata}
+        />
+      );
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    test('renders alert component with mail error', () => {
```

Comment:
let us use `it` as opposed to `test` for uniformity as used in previous test examples.

Reply:
Okay, let me update that

Reply:
Done